### PR TITLE
Fix for Issue #172 - Add arguments to helper functions (+ docstring edits)

### PR DESCRIPTION
**Pull request by Fislix**
Hi, I'm new to GitHub and this is my first pull request 😁. Please let me know if there is anything I could do better for the next time I create one.

Besides adding the username argument for two helper functions, as issue #172 addressed, I also made two tiny edits in the docstrings (excluding the edits regarding the new username argument):

1. **get_following function:** changed _"Retrieve the first x amount of users that follow an account."_ to _"Retrieve the first x amount of users that an account is following."_ (I thought the prior version sounded a bit mistakable. I didn't want to open a new issue for such a tiny change but if you don't like it, I'll, of course, change it back)
2. **unfollow_user function:** In the arguments-section, it said "user_id/username: the user_id/username of the account to **follow**" so I changed it to "... unfollow"

Let me know if you want me to change something,
thanks in advance

### CODE REVIEW

File: instauto/helpers/friendships.py [hunk 1]
```diff
@@ -10,20 +10,30 @@
...
     Args:
         client: your ApiClient
         user_id: the user_id of the account to retrieve followers from
+	    username: the username of the account to retrieve followers from
         limit: the maximum amount of followers to retrieve
 
     Returns:
         A list containing Instagram user objects (examples/objects/user.json).
     """
+    if user_id is not None and username is not None:
```
Comment (stanvanrooy) [inline instauto/helpers/friendships.py:28]: If both are provided, we just use the `user_id`. It's like that everywhere, so let's keep that standard.

File: instauto/helpers/friendships.py [hunk 2]
```diff
@@ -37,20 +47,30 @@ def get_followers(client: ApiClient, user_id: str, limit: int) -> typing.List[mo
...
     Args:
         client: your ApiClient
         user_id: the user_id of the account to retrieve following from
+	    username: the username of the account to retrieve following from
         limit: the maximum amount of users to retrieve
 
     Returns:
         A list containing Instagram user objects (examples/objects/user.json).
     """
+    if user_id is not None and username is not None:
```
Comment (stanvanrooy) [inline instauto/helpers/friendships.py:65]: Same as above.

**Review by stanvanrooy — CHANGES_REQUESTED** [review]
A few minor things. The PR looks good overall. Thanks!

File: instauto/helpers/friendships.py (hunk 1 above)
Comment (Fislix) [reply instauto/helpers/friendships.py:28]: Sure thing! I copied the argument checking logic from the functions further down in that file. Should I change it there too?

Comment (stanvanrooy) [reply instauto/helpers/friendships.py:28]: Aha, if you could :) Good catch

**Comment by stanvanrooy** [comment]
Looks good to me! Thanks for the contribution, Felix!

**Comment by stanvanrooy** [comment]
@all-contributors, please add @Fislix for code

**Comment by stanvanrooy** [comment]
@all-contributors please add @Fislix for code

**Pull request merged**